netmiko/ruckus/ruckus_fastiron.py

```diff
@@ -10,6 +10,17 @@
 class RuckusFastironBase(CiscoSSHConnection):
     """Ruckus FastIron aka ICX support."""
 
+    def _modify_connection_params(self) -> None:
+        """Modify connection parameters prior to SSH connection."""
+        paramiko_transport = getattr(paramiko, "Transport")
+        paramiko.Transport._preferred_kex.add(
+            "diffie-hellman-group14-sha1",
+            "diffie-hellman-group1-sha1",
+        )
+        paramiko.Transport._preferred_keys.add(
+            "ssh-rsa",
+        )
+
```

Comment:
Why do you retrieve `paramiko_transport`, but then change and reference `paramiko.Transport`?

Shouldn't it be:

```
        paramiko_transport = getattr(paramiko, "Transport")
        paramiko_transport._preferred_kex.add(
            "diffie-hellman-group14-sha1",
            "diffie-hellman-group1-sha1",
        )
        paramiko_transport._preferred_keys.add(
            "ssh-rsa",
        )
```